test/test_fake_tensor.py

```diff
@@ -580,6 +580,15 @@ def test_fake_parent_with_active_mode(self):
             a2 = mode2.from_tensor(a)
         self.assertTrue(free_symbols(a2.size(0)))
 
+    def test_fake_of_fake_constant(self):
+        mode1 = FakeTensorMode()
+        mode2 = FakeTensorMode(parent=mode1)
+        # Should not segfault
+        with mode2:
+            x = torch.tensor([2])
```

Comment:
assert `isinstance(x, FakeTensor) and x.constant is not None`? (mostly so someone can read the test and more easily understand what the correct behavior of FakeTensor + the `torch.Tensor` constructor is)